webpack.config.js

```diff
@@ -322,11 +322,21 @@ if ( ! config.isEnabled( 'desktop' ) ) {
 	);
 }
 
-// The SVG external content polyfill (svg4everybody) isn't needed for evergreen browsers, so don't bundle it.
+// List of polyfills that we skip including in the evergreen bundle.
+// CoreJS polyfills are automatically dropped using the browserslist definitions; no need to include them here.
+const polyfillsSkippedInEvergreen = [
+	// The fetch polyfill isn't needed for evergreen browsers, as they all support it.
+	/^isomorphic-fetch$/,
```

Comment:
want to land #32200 first and then rework this to use the same code?

Reply:
Sounds good! 👍

Reply:
Done!